src/components/forms/CreateSmsTemplate/server-action.ts

```diff
@@ -12,37 +10,23 @@ const $CreateSmsTemplateSchema = z.object({
   smsTemplateMessage: z
     .string({ message: 'Enter a template message' })
     .min(1, { message: 'Enter a template message' })
-    .max(MAX_SMS_CHARACTER_LENGTH, {
-      message: 'Template message too long',
-    }),
+    .max(918, { message: 'Template message too long' }),
```

Comment:
can we use the `MAX_SMS_CHARACTER_LENGTH` in the constants file please?

Reply:
updated